polaris-icons/rollup.config.js

```diff
@@ -203,6 +203,9 @@ function replaceFillAttributeSvgoPlugin() {
   };
 }
 
+/**
+ * @type {import('rollup').RollupOptions}
+ */
```

Comment:
Nitpick: you can specify these docblocks on one line

```suggestion
/** @type {import('rollup').RollupOptions} */
```